lib/pages/App.js

```diff
@@ -277,7 +277,7 @@ export class App extends Component {
       <div className={styles.footerRow}>
         <div className={styles.version}>{`v${version} beta`}</div>
         <div className="mailto">
-          <a className={styles.footerLink} href="mailto:support@tidepool.org?Subject=Feedback on Uploader" target="mailto">Get support</a>
+          <a className={styles.footerLink} href="http://support.tidepool.org/" target="_blank">Get support</a>
```

Comment:
@jebeck Will there be a secure -> insecure warning from this? Of course, https://support.tidepool.org also gives an certificate warning, so I hope not.